aksel.nav.no/website/pages/eksempler/radio/error.tsx

```diff
@@ -1,14 +1,21 @@
+import { useState } from "react";
 import { Radio, RadioGroup } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
 const Example = () => {
-  const handleChange = (val: any) => console.log(val);
+  const [error, setError] = useState(
+    "Du må velge en aldersgruppe før du kan gå videre.",
+  );
+  const handleChange = (val: any) => {
```

Comment:
`val` bør vel kunne types til `string`? (Har selvfølgelig ikke så mye å si for eksempelet, men tenker vi ikke trenger å oppfordre til å bruke `any`.)
```suggestion
  const handleChange = (val: string) => {
```